.editorconfig

```diff
@@ -0,0 +1,13 @@
+; editorconfig.org
+root = true
+charset= utf8
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
@joshredwards is this change intentional?

Reply:
It's just whats come from the `plugin-kit` setup

Reply:
Given V2 didn't have it, I'll remove this file.